java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -416,6 +416,43 @@ void testSliceWithArray() {
     }
   }
 
+  @Test
+  void testSplitWithArray() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    try(ColumnVector cv = ColumnVector.fromBoxedInts(10, 12, null, null, 18, 20, 22, 24, 26, 28)) {
+      Integer[][] expectedData = {
+          {10},
+          {12, null},
+          {null, 18},
+          {20, 22, 24, 26},
+          {28}};
+
+      ColumnVector[] splits = cv.split(1, 3, 5, 9);
+      try {
+        assertEquals(expectedData.length, splits.length);
+        for (int splitIndex = 0; splitIndex < splits.length; splitIndex++) {
+          ColumnVector subVec = splits[splitIndex];
+          subVec.ensureOnHost();
+          assertEquals(expectedData[splitIndex].length, subVec.getRowCount());
+          for (int subIndex = 0; subIndex < expectedData[splitIndex].length; subIndex++) {
```

Comment:
Curious why assertColumnsAreEqual was not leveraged for this test and instead walks each result column directly.

Reply:
I copied and pasted the test from `slice`.  If you want me to change it I am happy to.

Reply:
Oh, sorry, though this was a new test.  Would be nice to clean this up but not required.